src/module/flows/full-repair.ts

```diff
@@ -1,27 +1,49 @@
 // Import TypeScript modules
-import { prepareTextMacro } from "./text";
-import { LancerActor } from "../actor/lancer-actor";
+import { LANCER } from "../config";
+import { LancerItem } from "../item/lancer-item";
+import type { LancerActor } from "../actor/lancer-actor";
+import { renderTemplateStep } from "./_render";
+import { LancerFlowState } from "./interfaces";
+import { Flow, FlowState } from "./flow";
+import { UUIDRef } from "../source-template";
 
-export function prepareFullRepairMacro(actor_: string | LancerActor) {
-  // Determine which Actor to speak as
-  let actor = LancerActor.fromUuidSync(actor_);
+const lp = LANCER.log_prefix;
+
+export class FullRepairFlow extends Flow<LancerFlowState.TextRollData> {
+  constructor(uuid: UUIDRef | LancerItem | LancerActor, data?: Partial<LancerFlowState.TextRollData>) {
+    // Initialize data if not provided
+    const initialData: LancerFlowState.TextRollData = {
+      title: data?.title || "",
+      description: data?.description || "",
+      tags: data?.tags || [],
+    };
+
+    super("FullRepairFlow", uuid, initialData);
+    this.steps.set("displayFullRepairDialog", displayFullRepairDialog);
+    this.steps.set("executeFullRepair", executeFullRepair);
```

Comment:
Note that I made a couple changes that will need to be incorporated here. 
- The base `Flow` constructor no longer has the first argument for the flow name. The flow name is now defined at the class level.
- Steps are set at the class level as an array of strings, which are fetched from the step registry at runtime.
- Each flow source file has a function which registers its steps in the flow registry. Those functions are imported and called in `lancer.ts`.

You can see examples of this in `flows/attack.ts`, `flows.activation.ts`, etc... on the latest 2.0-rc commits. Should be fairly straightforward, mostly just copy-pasting things to a different place.

Reply:
Latest commit should fix all of these issues!